src/daemon/index.ts

```diff
@@ -57,3 +68,37 @@ async function checkJavaExtActivated(_context: vscode.ExtensionContext): Promise
 
    return true;
 }
+
+let corruptedCacheDetected: boolean = false;
+async function checkIfJavaServerCrashed(wait: number = 0/*ms*/) {
+   if (corruptedCacheDetected) {
+      return;
+   }
+
+   // wait Java Language Server to start
+   let count = 0;
+   while(count < wait) {
+      await delay(1000);
+      count += 1000;
+   }
+
+   const corruptedCache = !await daemon.processWatcher.start() && await daemon.logWatcher.checkIfWorkspaceCorrupted();
+   if (!corruptedCacheDetected && corruptedCache) {
+      corruptedCacheDetected = true;
+      sendInfo("", {
+         name: "corrupted-cache",
+      });
+      const ans = await vscode.window.showErrorMessage("A corrupted Java extension cache has been detected, please try to \"Clean Workspace\".",
```

Comment:
Should the quoted `Clean Workspace` be considered as a noun?

Reply:
I just reword the notification message as follows:

<img width="452" alt="image" src="https://user-images.githubusercontent.com/14052197/174706823-a8b39821-d54c-43ac-a3b8-2acbcf4fc0a7.png">